### ISSUE TITLE
invalid command 'bdist_wheel'         

### ISSUE THREAD

**Issue by huguesfontenelle** [unioslo/tsd-api-client#44]
```
error: invalid command 'bdist_wheel'            
Failed building wheel for pyyaml
Failed building wheel for s3cmd
```

It's missing a
```
pip3 install wheel
```

then
```
python3 setup.py bdist_wheel
```

Not sure if you want it in the `requirements.txt` or as an _install_requires_ in the `setup.py`.

**Comment by huguesfontenelle** [comment]
This has been fixed in v2.3.0 (or earlier).
Thank you, closing.

**Comment by huguesfontenelle** [comment]
This was with python3.5 , then I'd run into other bugs such as #48 .

But with python3.7 it works fine, without `wheel`.
